sdk/attestation/attestation/src/models/attestationToken.ts

```diff
@@ -168,7 +182,7 @@ export class AttestationToken {
     if (options.validationCallback !== undefined) {
       // If there is a validation error, the validationCallback will throw a customer
       // defined exception.
-      options.validationCallback(this, foundSigner ?? null);
+      options.validationCallback(this, foundSigner ?? undefined);
```

Comment:
I guess this is a fallout from the conversion to null, if `foundSigner` can be `undefined` you do not have to convert it to `undefined` explicitly.

Reply:
Changing from null to undefined :).

Reply:
Oh sorry I was confused by the original line because you were sending a potential null to a param that does not expect nulls. I see that you have `let foundSigner: AttestationSigner | null = null;` up there. I would say convert that to `let foundSigner: AttestationSigner | undefined = undefined;` instead and replace nulls with undefined.

For this call, you can just pass `passSigner` and you do not have to `??` it.

Reply:
Already done :). I just hadn't pushed the changes :). I'm pretty sure the current code is what you are suggesting.

Reply:
Yes almost! now you can just remove the `?? undefined` part here.

Reply:
I'm confused. My version of the code has `options.validationCallback(this, foundSigner);`. I don't see ?? undefined there.

Reply:
Ah - if you look at the current revision it doesn't have the `?? undefined`.